app/code/Magento/Backend/Console/Command/CacheCleanCommand.php

```diff
@@ -32,12 +32,15 @@ protected function configure()
      */
     protected function performAction(array $cacheTypes)
     {
-        $this->eventManager->dispatch('adminhtml_cache_flush_system');
+        if ($cacheTypes === [] || in_array('full_page', $cacheTypes)) {
```

Comment:
I believe you can use \Magento\PageCache\Model\Cache\Type::TYPE_IDENTIFIER here

Reply:
I know, and that's what I did first: https://github.com/magento/magento2/compare/01b8366baffebebb72e206fa2fff54ec384de08d..53b87ab003e20974497f3d450606030a93edd335 ;)

See my remark in the description, it requires adding a composer dependency between those two modules, and I'm not sure if that's worth it in this case...

Reply:
Oh, you're right, and to be honest I'm pretty sure there is approximately 0 chance on 100 that this value could change any day